server/src/main/java/org/candlepin/model/Owner.java

```diff
@@ -302,6 +326,10 @@ public boolean equals(Object obj) {
             !this.autobindDisabled.equals(other.autobindDisabled)) {
             return false;
         }
+        if ((this.contentAccessMode == null) ? (other.contentAccessMode != null) :
```

Comment:
This is a little too complicated for my tiny mind.  A ternary expression inside an if?  I didn't even know you could do that.  Can we simply this some?  Maybe

```java
if (contentAccessMode == null) {
  return other.contentAccessMode == null;
} else {
  return contentAccessMode.equals(other.contentAccessMode);
}
```